packages/@okta/vuepress-site/guides/sign-into-spa/section09/react/allowanon.md

```diff
@@ -1 +1 @@
-placeholder
\ No newline at end of file
+By default, all routes not defined using the `SecureRoute` will be accessible to unauthenticated users.
```

Comment:
By default, all routes not defined using the `SecureRoute` are accessible to unauthenticated users.